cuelang/podinfo/linkerd.cue

```diff
@@ -0,0 +1,23 @@
+package podinfo
+
+#ServiceProfile: {
```

Comment:
Let's remove this for now, the profile could be generated from podinfo API spec using CUE, but we can do this in another PR.